pyproject.toml

```diff
@@ -45,7 +45,8 @@ dev = [
     "pylint",
     "pytest",
     "pytest-cov",
-    "pydantic", # Used in unit tests
+    "pydantic>2", # Used in unit tests
+    "numba",
```

Comment:
- I have the feeling that the test case is simple enough to justify the old `parse_obj_as` code?
- Are we doing any `jit` already?

Reply:
1. `parse_obj_as` is officially deprecated since Pydantic v2. With v3 likely coming next spring or early summer (see also https://docs.pydantic.dev/latest/version-policy/#pydantic-v3-and-beyond). since it just involves `dev` dependencies, living at HEAD is OK
2. `numba` was proposed by `pandapower` for the validation tests. it produced a warning and i saw no reason to keep the warning if this was all that was necessary to fix it and it's just a `dev` dependency